src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -625,7 +627,9 @@ private CustomResourceSnapshot ToSnapshot(Container container, CustomResourceSna
                 new(KnownProperties.Container.Ports, GetPorts()),
             ],
             EnvironmentVariables = environment,
-            CreationTimeStamp = container.Metadata.CreationTimestamp?.ToLocalTime(),
+            CreationTimeStamp = container.Metadata.CreationTimestamp?.ToUniversalTime(),
```

Comment:
@mitchdenny @davidfowl Why are you converting to local time here? My experience is timestamps are UTC. Values are always converted to UTC when sent to the dashboard and the local browser time is used to convert it to a local time. Keeping it UTC up until that point seems good, but maybe there is a reason that I don't see.

Reply:
I don't recall who wrote this

Reply:
Mitch couldn't remember either.

I haven't seen any regression from changing it.